pandas/tests/series/test_analytics.py

```diff
@@ -237,6 +238,59 @@ def test_npdiff(self):
         r = np.diff(s)
         assert_series_equal(Series([nan, 0, 0, 0, nan]), r)
 
+    def test_dt_nm_bool_diff(self):
```

Comment:
bonus point if you can parameterize this test

Reply:
Thanks for the offer, but for now this will have to do. I basically added 3 lines of code that should have been added more than a few years ago. The need for writing a few tests I can get behind, but I would rather not completely refactor the whole file. The rest is implemented.